grails-app/views/deliveryNote/_printPage.gsp

```diff
@@ -4,6 +4,18 @@
         <thead>
             <tr>
                 <th><warehouse:message code="report.number.label"/></th>
+                <g:each in="${requisitionItems?.sort()}" status="i" var="requisitionItem">
```

Comment:
I think you could use find (http://grails.asia/groovy-find), both for level1 and level2. IMHO there is no need for this iterator here.